devel/JSONModel/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               xcode 1.0
+PortGroup               github 1.0
+
+github.setup            JSONModel JSONModel cd940e7
+version                 1.7.0-20170324
+categories              devel
+platforms               darwin
+license                 MIT
+maintainers             {@janosch mailbox.org:janosch1} openmaintainer
+description             ${name} allows rapid creation of smart data models.
+long_description        ${name} allows rapid creation of smart data models. \
+                        You can use it in your iOS, macOS, watchOS and tvOS apps. \
+                        Automatic introspection of your model classes and JSON \
+                        input drastically reduces the amount of code you have to write.
+
+checksums               rmd160  82130455c3df91ad2735737ab75f153bade91075 \
+                        sha256  93afb12793e17678b8a886216730c4b4432ffecf89ef78ed415c9c00a662c149 \
+                        size    95251
+
+xcode.scheme            JSONModel-mac
+xcode.target            JSONModel-mac
+xcode.configuration     Release
+xcode.destroot.type     framework
+
+xcode.destroot.settings CODE_SIGN_STYLE=Manual \
+                        CODE_SIGN_IDENTITY=- \
+                        ENABLE_HARDENED_RUNTIME=NO \
+                        SKIP_INSTALL=NO \
+                        INSTALL_DIR=${destroot}${frameworks_dir} \
+                        LD_DYLIB_INSTALL_NAME=${frameworks_dir}/${name}.framework/Versions/A/${name}
+
+# xcode PortGroup sets the DerivedData to /var/root/... which is not writeable. This fixes it.
```

Comment:
The xcode portgroup does not set the derived data path at all. Rather, Xcode has a bug where it sets the derived data path based on the user's home directory rather than the $HOME environment variable; the macports user's home directory is not writable. The xcode portgroup *should* [set the derived data path to a writable location](https://trac.macports.org/ticket/57137). For some reason, nobody has fixed that and everybody instead adds workarounds to individual ports.